tests/dragonfly/replication_test.py

```diff
@@ -1024,23 +1024,23 @@ async def test_flushall_in_full_sync(df_local_factory, df_seeder_factory):
     c_replica = aioredis.Redis(port=replica.port)
     await c_replica.execute_command(f"REPLICAOF localhost {master.port}")
 
-    async def get_sync_mode(c_replica):
-        result = await c_replica.execute_command("role")
-        return result[3]
+    async def get_sync_mode(c_master):
+        result = await c_master.execute_command("role")
+        return result[1][0][2]
```

Comment:
Consider adding a small comment detailing which array has what
Not a must though